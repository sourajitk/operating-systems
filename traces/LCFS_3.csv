2,0,1
4,0,2
6,0,3
8,10,1
10,10,2
12,10,3
14,10,4
16,20,1
18,20,2
20,20,3
