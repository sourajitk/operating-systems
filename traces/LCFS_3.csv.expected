2,1
4,6
6,4
8,11
10,20
12,18
14,15
16,21
18,26
20,24
